packages/x-adapter-platform/src/platform.adapter.ts

```diff
@@ -1,18 +1,24 @@
-import { searchEndpointAdapter } from './endpoint-adapters/search.endpoint-adapter';
-import { PlatformAdapter } from './types/platform-adapter.types';
-import { empathizeEndpointAdapter } from './endpoint-adapters/empathize.endpoint-adapter';
-import { topClickedEndpointAdapter } from './endpoint-adapters/top-clicked.endpoint-adapter';
+// eslint-disable-next-line max-len
+import { identifierResultsEndpointAdapter } from './endpoint-adapters/identifier-results.endpoint-adapter';
 import { nextQueriesEndpointAdapter } from './endpoint-adapters/next-queries.endpoint-adapter';
+// eslint-disable-next-line max-len
+import { popularSearchesEndpointAdapter } from './endpoint-adapters/popular-searches.endpoint-adapter';
+// eslint-disable-next-line max-len
+import { querySuggestionsEndpointAdapter } from './endpoint-adapters/query-suggestions.endpoint-adapter';
+// eslint-disable-next-line max-len
+import { recommendationsEndpointAdapter } from './endpoint-adapters/recommendations.endpoint-adapter';
 import { relatedTagsEndpointAdapter } from './endpoint-adapters/related-tags.endpoint-adapter';
-import { skuSearchEndpointAdapter } from './endpoint-adapters/sku-search.endpoint-adapter';
+import { searchEndpointAdapter } from './endpoint-adapters/search.endpoint-adapter';
 import { taggingEndpointAdapter } from './endpoint-adapters/tagging.endpoint-adapter';
+import { PlatformAdapter } from './types/platform-adapter.types';
```

Comment:
No, because that can cause `circular dependencies`. We should avoid `importing` from `barrel` files inside the same `package`. `Barrel` files are only useful when `importing` `modules` from another `package`.